core/admin/admin.py

```
@@ -0,0 +1,346 @@
+class AdminUserAdmin(ModelView, model=AdminUser):
+    """Admin view for Admin Users"""
+    name = "Admin User"
+    name_plural = "Admin Users"
+    icon = "fa-solid fa-user-shield"
+    
+    column_list = [
+        AdminUser.id,
+        AdminUser.username,
+        AdminUser.is_super_admin,
+        AdminUser.created_at,
+    ]
```

Ruff (0.14.10)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)